es/news/_posts/2016-02-24-support-plan-of-ruby-2-0-0-and-2-1.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: news_post
+title: "Planes de soporte para Ruby 2.0.0 y Ruby 2.1"
```

Comment:
Please, update this: `Planes de soporte para Ruby 2.0.0 y Ruby 2.1` to `Planes para el soporte de Ruby 2.0.0 y Ruby 2.1`